indico/modules/categories/models/categories.py

```diff
@@ -218,6 +218,10 @@ def is_root(cls):
         return cls.parent_id.is_(None)
 
     @property
+    def is_empty(self):
+        return not(self.deep_children_count or self.deep_events_count)
```

Comment:
space after `not` since it's not a function. and  IMO demorganizing it would look better:

    return not self.deep_children_count and not self.deep_events_count